db/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self) -> models.CharField:
+        return self.name
+
+
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self) -> models.CharField:
```

Comment:
sorry haven't noticed.

wrong type annotation. this method returns str

Reply:
done, buy i already received comment about this:

You should place a return statement, not print in __str__. Indicate this in type annotation and do not overuse the f-strings, there is no need for them when you return CharField directly.